radixdlt-engine/src/main/java/com/radixdlt/atom/TxErrorCode.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.atom;
+
+public enum TxErrorCode {
```

Comment:
Would be great if we could catch these? Some might not be applicable...

* Amount not multiple of granularity? 
* Message longer than 255 bytes?
* Network of AccountAddress mismatches with active network? (got `"rdx1.."` but expected `"brx1.."` and vice versa)
* Network of ValidatorAddress mismatches with active network? (got `"vb1.."` but expected `"vr1.."` and vice versa)
* Network of RRI mismatches with active network? (got `".._rb1.."` but expected `".._rr1.."` and vice versa)

Reply:
granularity is not relevant I guess. for others I'll try to add appropriate handling